google/cloud/datastore_admin_v1/services/datastore_admin/client.py

```diff
@@ -652,7 +652,7 @@ def import_entities(
 
         .. code-block:: python
 
-            from google.cloud import datastore_admin_v1
+                    from google.cloud import datastore_admin_v1
 
             def sample_import_entities():
                 # Create a client
```

Comment:
Wait for the next gapic-generator-python release which will fix the formatting issue here.